schemas/input/csl-data.json

```diff
@@ -130,12 +154,30 @@
                     "$ref": "#/definitions/name-variable"
                 }
             },
+            "organizer": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/name-variable"
+                }
+            },
             "original-author": {
                 "type": "array",
                 "items": {
                     "$ref": "#/definitions/name-variable"
                 }
             },
+            "producer": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/name-variable"
+                }
+            },
+            "performer": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/name-variable"
+                }
+            },
```

Comment:
(minor nitpick, but "performer" and "producer" are not listed alphabetically)

Reply:
I'll fix these while I'm at it.